EasyPost.Tests/EnumTest.cs

```diff
@@ -1,12 +1,14 @@
 using System.Collections.Generic;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Diagnostics.CodeAnalysis;
+using EasyPost.Models.API;
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
     public class EnumTest
     {
-        [TestMethod]
+        [Fact]
+        [SuppressMessage("Assertions", "xUnit2017:Do not use Contains() to check if a value exists in a collection")]
```

Comment:
Issue: Don't suppress errors and sweep them under the rug when there are reliable ways to fix the underlying problem. 

Something like this maybe? https://www.pmichaels.net/tag/do-not-use-enumerable-any-to-check-if-a-value-exists-in-a-collection/